### ISSUE TITLE
Text paste and Undo/Redo

### ISSUE THREAD

**Issue by georgtree** [StefanSchippers/xschem#256]
Hello!
A few things I noticed while editing text fields:
1) Paste with selection doesn't replace selection in auxiliary text fields, as it does in main text field, for example:
![image](https://github.com/user-attachments/assets/c6ee9ce8-c179-4a54-a864-5f27985a0e84)

2) It would be great to have undo and redo working in properties dialogs with text.

Thank you in advance.

**Comment by StefanSchippers** [comment]
In last commit the entry widgets (= one line texts) selection is deleted when pasting.
there is no global option in tcl-tk for doing  that, so I had to add a <<Paste>> binding that deletes selection (if any) to all xschem entry widgets.

I have also added the -undo option to all multiline text widgets.
Single line entry widgets do not have an undo mechanism.
Tcl-tk default bindings: Ctrl-Z does Undo, Ctr-Shift-Z does Redo.
